src/TiciTacaToeyGameEngine.ts

```diff
@@ -248,6 +270,10 @@ class TiciTacaToeyGameEngine implements GameEngine {
         const gameReadyToStart =
           updatedPlayersList.length === this.games[gameId].playerCount;
 
+        if (gameReadyToStart) {
+          this.games[gameId].timers[firstPlayer].start();
```

Comment:
This should not use the `firstPlayer` variable, instead extract the logic on line 283 out and use here and on line 283 as well.